wpilibc/src/main/native/include/frc/PIDController.h

```diff
@@ -45,9 +45,9 @@ class PIDController : public PIDBase, public Controller {
    * @param Kd     the derivative coefficient
    * @param source The PIDSource object that is used to get values
    * @param output The PIDOutput object that is set to the output value
-   * @param period the loop time for doing calculations. This particularly
-   *               effects calculations of the integral and differental terms.
-   *               The default is 50ms.
+   * @param period the loop time for doing calculations in seconds. This
+   *               particularly effects calculations of the integral and
```

Comment:
Heh, how long have the docs been using the wrong spelling for `affects`?

Reply:
Thanks, I'll update that as well.